COMPATIBILITY.md

```diff
@@ -69,4 +69,4 @@ For cases when a first-class Workflow step (rather than an adaptation of functio
 
 - [X] `parallel-test-executor`: supported with `splitTests` step since 1.6
 - [ ] `gerrit-trigger`: [JENKINS-26102](https://issues.jenkins-ci.org/browse/JENKINS-26102), [JENKINS-26103](https://issues.jenkins-ci.org/browse/JENKINS-26103)
-- [ ] `mailer`: [JENKINS-26104](https://issues.jenkins-ci.org/browse/JENKINS-26104)
+- [X] `mailer` (`workflow`): supported with `mail` step since 1.2
```

Comment:
this is confusing as mailer plugin 1.2 is very old.  THis should say supported since 1.2 of *workflow* (since all other sinces refer to the plugin version not workflow version.